spec/integration/issues/issue-1027-sheet-defined-data-validate.spec.js

```diff
@@ -0,0 +1,27 @@
+const ExcelJS = verquire('exceljs');
+
+// this file to contain integration tests created from github issues
+const TEST_XLSX_FILE_NAME = './spec/out/wb.test.xlsx';
+
+describe('github issues', () => {
+  it('issue 1027 - Broken due to Cannot set property \'marked\' of undefined error', () => {
+    const wb = new ExcelJS.Workbook();
+    const ws = wb.addWorksheet('Sheet1');
+
+    const range = 'A2:A1048576';
+
+    ws.dataValidations.model[range] = {
+      allowBlank: true,
+      error: 'Please use the drop down to select a valid value',
+      errorTitle: 'Invalid Selection',
+      formulae: ['"Apples,Bananas,Oranges"'],
+      showErrorMessage: true,
+      type: 'list',
+    };
+
+    return wb.xlsx.writeFile(TEST_XLSX_FILE_NAME).then(() => {
+      // arriving here is success
+      expect(true).to.equal(true);
+    });
```

Comment:
Please change this to just 
```js
await wb.xlsx.writeFile(TEST_XLSX_FILE_NAME);
// arriving here is success
```

Reply:
Since none of the other test are using await / async, I ended up just doing: `return wb.xlsx.writeFile(TEST_XLSX_FILE_NAME);`